app/src/main/res/layout/fragment_embark_address_auto_complete_action.xml

```diff
@@ -15,54 +16,29 @@
         tools:itemCount="3"
         tools:listitem="@layout/embark_message_item" />
 
-    <TextView
-        android:id="@+id/response"
+    <include
+        android:id="@+id/responseContainer"
+        layout="@layout/embark_response"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/base_margin_double"
-        android:layout_marginTop="@dimen/base_margin_half"
-        android:layout_marginEnd="@dimen/base_margin_double"
-        android:alpha="0"
-        android:textAppearance="?textAppearanceSubtitle1"
-        android:theme="@style/ThemeOverlay.Hedvig.Embark.Chat.Self"
-        android:translationY="100dp"
-        android:visibility="gone"
+        android:layout_marginHorizontal="@dimen/base_margin_double"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/messages" />
+        app:layout_constraintTop_toBottomOf="@id/messages"
+        app:layout_constraintWidth_max="@dimen/embark_chat_bubble_max_width" />
 
-    <com.google.android.material.card.MaterialCardView
-        android:id="@+id/input_layout"
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/input_card"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginStart="@dimen/base_margin_double"
         android:layout_marginEnd="@dimen/base_margin_double"
         android:layout_marginBottom="@dimen/base_margin_double"
+        android:transitionGroup="true"
```

Comment:
This was a very tricky thing. When under a material transition, ComposeView items just make the app crash as reported here as well https://issuetracker.google.com/issues/206947893 This fixes it.